src/ApiService/ApiService/onefuzzlib/Containers.cs

```diff
@@ -108,17 +129,48 @@ public class Containers : IContainers {
         return (start, expiry);
     }
 
-    public async System.Threading.Tasks.Task saveBlob(Container container, string name, string data, StorageType storageType) {
+    public async Async.Task SaveBlob(Container container, string name, string data, StorageType storageType) {
         var client = await FindContainer(container, storageType) ?? throw new Exception($"unable to find container: {container.ContainerName} - {storageType}");
-
         await client.UploadBlobAsync(name, new BinaryData(data));
     }
 
+    //TODO: get this ones on startup and cache (and make this method un-accessible to everyone else)

```

Comment:
Did you want to make GetInstanceId private?

Reply:
that's for later